spec/classes/foreman_proxy__plugin__pulp_spec.rb

```diff
@@ -2,8 +2,16 @@
 
 describe 'foreman_proxy::plugin::pulp' do
 
+  let :puppet_environmentpath do
+    if Gem::Version.new(Puppet.version) >= Gem::Version.new('4.0')
+      {:puppet_environmentpath => '/etc/puppetlabs/code/environments'}
+    else
+      {}
```

Comment:
Actually the fact exists puppet 3, it's just empty. I'm not how pick handles empty strings so it's a good thing to test for here.